client/landing/gutenboarding/index.tsx

```diff
@@ -134,26 +134,24 @@ function waitForSelectedSite(): Promise< Site | undefined > {
 	} ).finally( unsubscribe );
 }
 /**
- * If available-designs-config.json has been updated, replace the cached list
- * of designs with the updated designs
+ * If the list of available designs (stored in the `@automattic/domain-picker` package)
+ * has been updated, replace the cached list of designs with the updated designs.
  */
 function ensureRandomizedDesignsAreUpToDate() {
 	const designsInStore = select( ONBOARD_STORE ).getRandomizedDesigns();
+	const availableDesigns = getAvailableDesigns();
 	if ( ! isDeepEqual( designsInStore.featured, availableDesigns.featured ) ) {
-		dispatch( ONBOARD_STORE ).setRandomizedDesigns( {
-			...availableDesigns,
-			featured: shuffle( availableDesigns.featured ),
-		} );
+		dispatch( ONBOARD_STORE ).setRandomizedDesigns( getAvailableDesigns( { randomize: true } ) );
 	}
 }
 
 /**
  *
- * Compare cached designs in the ONBOARD_STORE to the source of designs in
- * available-designs-config.json
+ * Compare cached designs in the ONBOARD_STORE to the source of designs defined
+ * in the `@automattic/domain-picker` package
  *
  * @param stored randomizedDesigns cached in WP_ONBOARD
- * @param available designs sourced from available-designs-config.json
+ * @param available designs sourced from the `@automattic/domain-picker` package
```

Comment:
s/domain/design/

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/51633/commits/f923bdbf308024aa413ed864687b6237abb8dddb